website/docs/cloud-docs/api-docs/oauth-clients.mdx

```diff
@@ -218,11 +230,13 @@ Properties without a default value are required.
 | `data.attributes.name`               | string | `null`  | An optional display name for the OAuth Client. If left `null`, the UI will default to the display name of the VCS provider.                                                                    |
 | `data.attributes.key`                | string |         | The OAuth Client key. It can refer to a Consumer Key, Application Key, or another type of client key for the VCS provider.                                                                     |
 | `data.attributes.http-url`           | string |         | The homepage of your VCS provider (e.g. `"https://github.com"` or `"https://ghe.example.com"` or `"https://gitlab.com"`).                                                                      |
-| `data.attributes.api-url`            | string |         | The base URL as per your VCS provider's API documentation (e.g. `"https://api.github.com"`, `"https://ghe.example.com/api/v3"` or `"https://gitlab.com/api/v4"`).                            |
+| `data.attributes.api-url`            | string |         | The base URL as per your VCS provider's API documentation (e.g. `"https://api.github.com"`, `"https://ghe.example.com/api/v3"` or `"https://gitlab.com/api/v4"`).                              |
```

Comment:
looks like an existing typo with the dangling `or..`, but can we go ahead and fix it in this PR?